CHANGELOG.md

```diff
@@ -33,6 +34,29 @@
 - [v0.5.0](#v050---2020-08-06)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v0.29.1 - 2023-04-18
+
+### Release notes
```

Comment:
You can remove unused headers, else it looks good to me.

Reply:
Removed them just now
Thx